README.md

```diff
@@ -1,403 +1,42 @@
 
 ```bash
-bin/rails s
+$ bundle exec rails g solidus:views:override --only products/show
 ```
 
-The [`solidus_frontend`](https://github.com/solidusio/solidus/tree/master/frontend) storefront will be accessible at [http://localhost:3000/](http://localhost:3000/)
-and the admin can be found at [http://localhost:3000/admin/](http://localhost:3000/admin/).
-
-For information on how to customize your store, check out the [customization guides](https://guides.solidus.io/developers/customizations/overview.html).
-
-### Default Username/Password
-
-As part of running the above installation steps, you will be asked to set an admin email/password combination. The default values are `admin@example.com` and `test123`, respectively.
-
-### Questions?
-
-The best way to ask questions is to [join the Solidus Slack](http://slack.solidus.io/) and join the [#support channel](https://solidusio.slack.com/messages/support/details/).
-
-## Installation options
-
-Instead of a stable build, if you want to use the bleeding edge version of
-Solidus, use this line:
-
-```ruby
-gem 'solidus', github: 'solidusio/solidus'
-```
-
-**Note: The master branch is not guaranteed to ever be in a fully functioning
-state. It is too risky to use this branch in production.**
-
-By default, the installation generator (`solidus:install`) will run
-migrations as well as adding seed and sample data. This can be disabled using
+The argument to `--only` can also be a substring of the name of the view from the `app/views/spree` folder:
 
 ```bash
-bin/rails g solidus:install --migrate=false --sample=false --seed=false
+$ bundle exec rails g solidus:views:override --only product
 ```
 
-You can always perform any of these steps later by using these commands.
+This will copy all views whose directory or filename contains the string "product".
 
-```bash
-bin/rails railties:install:migrations
-bin/rails db:migrate
-bin/rails db:seed
-bin/rails spree_sample:load
-```
+### Handle upgrades
 
-There are also options and rake tasks provided by
-[solidus\_auth\_devise](https://github.com/solidusio/solidus_auth_devise).
+After upgrading Solidus to a new version run the generator again and follow on screen instructions.
 
-## Performance
-
-You may notice that your Solidus store runs slowly in development mode. This
-can be because in development each CSS and JavaScript is loaded as a separate
-include. This can be disabled by adding the following to
-`config/environments/development.rb`.
-
-```ruby
-config.assets.debug = false
-```
-
-### Turbolinks
-
-To gain some extra speed you may enable Turbolinks inside of Solidus admin.
-
-Add `gem 'turbolinks', '~> 5.0.0'` into your `Gemfile` (if not already present)
-and change `vendor/assets/javascripts/spree/backend/all.js` as follows:
-
-```js
-//= require turbolinks
-//
-// ... current file content
-//
-//= require spree/backend/turbolinks-integration.js
-```
+## Testing
```

Comment:
We're not adding tests to the host application if I'm not wrong. So I think this section should instead be called "Contributing".